zerver/tests/webhooks/test_hellosign.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from typing import Text
+from zerver.lib.test_classes import WebhookTestCase
+
+class HelloSignHookTests(WebhookTestCase):
+    STREAM_NAME = 'hellosign'
+    URL_TEMPLATE = "/api/v1/external/hellosign?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'hellosign'
+
+    # Note: Include a test function per each distinct message condition your integration supports
```

Comment:
You haven't tested the full functionality of the combinations (`a, b and c`).

Also, this comment isn't necessary?